src/Glpi/Form/Comment.php

```diff
@@ -141,7 +142,17 @@ public function displayBlockForEditor(): void
      */
     public function getForm(): Form
     {
-        return $this->getItem()->getItem();
+        $section = $this->getItem();
+        if (!$section || !($section instanceof Section)) {
```

Comment:
`null instanceof MyClass` will return `false`, no need to do an additional check.

```suggestion
        if (!($section instanceof Section)) {
```